pygac/reader.py

```diff
@@ -723,6 +723,21 @@ def get_tle_lines(self):
         self.tle_lines = tle1, tle2
         return tle1, tle2
 
+    def get_sat_angles_without_tle(self):
+        """Get satellite angles using lat/lon from data to approximate satellite postition instead of TLE."""
+        from pyorbital.orbital import get_observer_look as get_observer_look_no_tle
+        sat_alt = 850.0  # km  TIROS-N OSCAR
+        mid_column = int(0.5*self.lons.shape[1])
+        sat_azi, sat_elev = get_observer_look_no_tle(
+            self.lons[:, mid_column][:, np.newaxis],
+            self.lats[:, mid_column][:, np.newaxis],  # approximate satellite position
+            sat_alt,  # approximate satellite altitude
+            self.times[:, np.newaxis],
+            self.lons, self.lats, 0)
+        # Sometimes the get_observer_look_not_tle returns nodata instead of 90.
+        sat_elev[:, mid_column] = 90
```

Comment:
Since `pyorbital` is part of the PyTroll libraries, I would prefer to fix the issue there instead of fixing the library output in `pygac`. The issue is very likely a problem with the defined domain of trigonometric functions and their inverse, my candidates are [arctan](https://github.com/pytroll/pyorbital/blob/f1e50fcee10ef85688c23c78c9ebae26956db805/pyorbital/orbital.py#L126) and [arcsin](https://github.com/pytroll/pyorbital/blob/f1e50fcee10ef85688c23c78c9ebae26956db805/pyorbital/orbital.py#L146).

Reply:
Absolutely! But with the fix here it is ok to use also an old version of pyorbital. I think that is nice. I have made an issue to pyorbital https://github.com/pytroll/pyorbital/issues/72.

Reply:
I was thinking whether that correction could be applied only if the pyorbital version <= current version. This would remind us to remove that fix in the future. On the other hand we don't know whether the issue will be solved in the next pyorbital release. So what about referencing the github issue in the comment?